python/hail/linalg/blockmatrix.py

```diff
@@ -707,8 +925,41 @@ def entries(self):
         :class:`.Table`
             Table with a row for each entry.
         """
-        hc = Env.hc()
-        return Table(self._jbm.entriesTable(hc._jhc))
+        return Table(self._jbm.entriesTable(Env.hc()._jhc))
 
 
 block_matrix_type.set(BlockMatrix)
+
+
+def _ndarray_as_matrix_float64(nd):
```

Comment:
"matrix" is a bit overloaded in this code.  (Numpy?  Block?  Breeze?)  You might clarify.